### PR TITLE
feat(userevent): add agree photo field to user event

### PR DESCRIPTION
**Pull request by henriettebs**
# PR NAME
Lint, test and build
## Description
Link to Trello task: ...

Comments/issues/screenshots:
#204 

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [x] PR name is descriptive
- [ ] The PR includes the link to the relevant Trello task
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)

- [ ] `yarn build` was run locally without errors
- [ ] `yarn test` has passed locally and any fixes were made for failures
- [ ] `yarn format` was run locally and fixes were made for new errors

## Pull request type

Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

### CODE REVIEW

File: src/containers/EventAdministration/components/EventParticipant.js [hunk 1]
```diff
@@ -101,6 +101,7 @@ const EventParticipant = (props) => {
         <Typography>Studie: {getUserStudyShort(userInfo.user_study)}</Typography>
         <Typography>Årstrinn: {userInfo.user_class}. klasse</Typography>
         {userInfo.allergy !== '' && <Typography>Allergier: {userInfo.allergy}</Typography>}
+        {user.agree_photo && <Typography>Vil ikke bli tatt bilde av</Typography>}
```
Comment (olros) [inline src/containers/EventAdministration/components/EventParticipant.js:104]: Vi gjorde en feil her. "Vil ikke bli tatt bilde av" skal vises hvis `agree_photo` er `false` så da må vi endre `user.agree_photo` til `!user.agree_photo`

**Review by olros — CHANGES_REQUESTED** [review]

File: src/containers/EventDetails/components/EventDialog.js [hunk 2]
```diff
@@ -308,6 +309,10 @@ const EventDialog = (props) => {
                 {optionalFieldsAnswers.length > 0 && (
                   <Questions classes={classes} handleOptionalFieldInput={handleOptionalFieldInput} optionalFieldsAnswers={optionalFieldsAnswers} />
                 )}
+                <FormControlLabel
+                  control={<Checkbox checked={agreePhoto} onChange={(e) => setAgreePhoto(e.target.checked)} />}
+                  label='Jeg godtar å bli tatt bilde av på arrangementet'
```
Comment (Zenjjim) [inline src/containers/EventDetails/components/EventDialog.js:314]: Jeg godtar at bilder av meg kan deles på TIHLDE sine plattformer

**Review by olros — APPROVED** [review]
Ser bra ut! 💯 

For å merge dette inn i dev-branchen vil jeg gjerne at du bruker "Squash and merge" slik at commitsene i denne branchen samles til 1 commit i dev:
image

Mulig du må trykke på "Update branch" først slik at branchen er oppdatert mot dev først.

**Comment by Zenjjim** [comment]
Er dette koblet opp til backend?

**Comment by Zenjjim** [comment]
feltet i backend heter `allow_photo`,så og er i prod så bare å koble seg opp :D

**Comment by olros** [comment]
> Er dette koblet opp til backend? @henriettebs @olros 

Avhenger av tihlde/API#128 som du nettop merget ja

**Pull request merged**